src/main/resources/assets/utilitiesinexcess/lang/en_US.lang

```diff
@@ -103,191 +124,253 @@ chat.inversion_ritual.darkness= - There is no light: %s
+nei.infopage.uie.smart_pump.2=Before pumping, it will try to output fluid to tanks located directly above it. The pump works chunk-wise and works in a diamond with a radius of 5 chunks. During operation, the pump will keep its own chunk loaded as well as the current working chunk.
 
 tile.wood_spike.name=Wooden Spikes
 tile.iron_spike.name=Iron Spikes
 tile.gold_spike.name=Gold Spikes
 tile.diamond_spike.name=Diamond Spikes
-tile.wood_spike.desc=Will leave enemies at the brink of death
-tile.iron_spike.desc=Enemies will drop regular loot
-tile.gold_spike.desc=Enemies will drop regular loot and experience
-tile.diamond_spike.desc=Enemies will drop "player-only" loot and experience
+
+nei.infopage.uie.woodSpike.1=These spikes will damage anything that walks on them, but will not kill them, instead leaving them at the minimum health possible.
+nei.infopage.uie.ironSpike.1=Iron spikes will kill enemies that walk on them, dropping only their non-"player-only" loot.
+nei.infopage.uie.goldSpike.1=Gold spikes will kill enemies that walk on them, dropping their experience and non-"player-only" loot.
+nei.infopage.uie.diamondSpike.1=Diamond spikes will kill enemies that walk on them, dropping experience and "player-only" loot.
 
 item.heavenly_ring.name=Heavenly Ring
-item.heavenly_ring.desc=Grants creative flight
+nei.infopage.uie.heavenly_ring.1=Grants the bearer the power of creative flight while in the inventory.
+nei.infopage.uie.heavenly_ring.2=Grants the bearer the power of creative flight while equipped in a baubles slot.
 
 item.watering_can_basic.name=Basic Watering Can
 item.watering_can_advanced.name=Advanced Watering Can
 item.watering_can_elite.name=Elite Watering Can
 item.watering_can.activated=Activated (Toggle)
 item.watering_can.deactivated=Deactivated (Toggle)
-item.watering_can.desc.1=It hydrates and accelerates
-item.watering_can.desc.2=crop growth in a %dx%d area.
-item.watering_can.desc.3=Crouch and use to toggle
-item.watering_can.desc.4=automatic mode.
+item.watering_can.desc.1=Area: %dx%d
+
+nei.infopage.uie.watering_can.1=Hydrates farmland and accelerates crop growth. Crouch and use to toggle automatic mode.
 
 item.fire_battery.name=Fire Battery
 item.fire_battery.desc=RF %s / %s
+nei.infopage.uie.fire_battery.1=Can be charged with RF and charge can be expended to fuel a furnace.
 
 item.glove.name=Glove
-item.glove.desc.1=+2,147,483,647 Breeze Projection
-item.glove.desc.2=Allows you to click with an empty hand without
-item.glove.desc.3=needing to clear an inventory slot while held
-item.glove.desc.4=Can also be put in a baubles slot
-item.glove.desc.5=Press %s to activate.
-item.glove.desc.6=§oP.S. Breeze Projection does nothing.
+item.glove.desc.1=Press %s to activate.
+item.glove.desc.2=+2,147,483,647 Breeze Projection
```

Comment:
You swapped the numbers on these around but not the colors so now it looks like this

<img width="597" height="156" alt="Image" src="https://github.com/user-attachments/assets/f61ce3d0-90df-453d-b54c-a320b8ec43ea" />